content/capabilities/reference-capabilities.md

```diff
@@ -96,12 +96,37 @@ String box // A string box
 String tag // A string tag
 ```
 
-__What does it mean when a type doesn't specify a reference capability?__ It means you are using the _default_ reference capability for that type, which is defined along with the type. Here's an example from the standard library:
+__What does it mean when a type doesn't specify a reference capability?__ It means you are using the _default_ reference capability for that type, which is defined along with the type. 
+
+__So do I have to specify a reference capability when I define a type?__ Only if you want to. There are sensible defaults that most types will use. These are `ref` for classes, `val` for primitives (i.e. immutable references) and `tag` for actors.
+
+## How to create objects with different capabilities
+
+When you write a constructor. By default, that constructor will either create a new object with `ref` or `tag` as the capability. In the case of actors, the constructor will always create a `tag`. For classes, if defaults to `ref` but you can create with other capabilities. Let's take a look at an example:
 
 ```pony
-class val String
+class Foo
+  let x: U32
+
+  new val create(x': U32) =>
+    x = x'
 ```
 
-When we use a `String` we usually mean a string value, so we make `val` the default reference capability for `String`.
+Now when you call `Foo.create(1)`, you'll get a `Foo val` instead of `Foo ref`.
+You could do something like this:
```

Comment:
Maybe say "You could also do something like this:". But it's not clear why you'd do this. Maybe mention again here that this is to provide different caps from the constructor.

Reply:
addressed